docs/ciphers.rst

```diff
@@ -127,15 +124,15 @@ which uses the ChaCha and Poly1305 algorithms, and which is already
 implemented by the Chrome browser.  Mozilla has delayed recommending
 ``0xcc13`` over compatibility and standardization concerns, but is likely
 to recommend it in the future once these concerns have been addressed. At
-that point, the Let's Encrypt client would likely follow the Mozilla
-recommendations and favor the use of this ciphersuite as well.)
+that point, Certbot would likely follow the Mozilla recommendations and favor
+the use of this ciphersuite as well.)
 
 The Let's Encrypt project may deviate from the Mozilla recommendations
 in the future if good cause is shown and we believe our users'
 priorities would be well-served by doing so. In general, please address
 relevant proposals for changing priorities to the Mozilla security
-team first, before asking the Let's Encrypt project to change the
-client's priorities. The Mozilla security team is likely to have more
+team first, before asking the Let's Encrypt project or EFF to change
```

Comment:
This should just be "the Certbot developers" (rather than "the Let's Encrypt project or EFF").